src/Avalonia.Controls/DragDrop/DragDrop.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Avalonia.Interactivity;
+
+namespace Avalonia.Controls.DragDrop
+{
+    public sealed class DragDrop : AvaloniaObject
```

Comment:
It could be a  static class. It also doesn't need to inherit from AvaloniaObject.